DumpCementeryDB.sql

```diff
@@ -19,106 +19,106 @@
 DB Structure Goes Here...
 */
 
-/* ####### Create Sector ####### */
+/* ####### Create Sectors ####### */
 
-DROP TABLE IF EXISTS `sektor`;
-CREATE TABLE `sektor` (
-  `id` int(11) NOT NULL AUTO_INCREMENT
+DROP TABLE IF EXISTS `Sectors`;
+CREATE TABLE `Sectors` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  'number' varchar(15)
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
 
 
-/* ####### Create Pracownik ####### */
+/* ####### Create Employees ####### */
 
-DROP TABLE IF EXISTS `pracownik`;
-CREATE TABLE `pracownik` (
+DROP TABLE IF EXISTS `Employees`;
+CREATE TABLE `Employees` (
     `id` int(11) NOT NULL AUTO_INCREMENT,
-    `imie` varchar(60) DEFAULT NULL,
-    `nazwisko` varchar(60) DEFAULT NULL,
-    'data_zatrudnienia' DATE DEFAULT NULL,
-    'wynagrodzenie' DECIMAL(7,5) DEFAULT NULL,
-    'id_zlecenia' INT DEFAULT NULL,
-    'id_kwatery' INT DEFAULT NULL
+    `first_name` varchar(60) DEFAULT NULL,
+    `last_name` varchar(60) DEFAULT NULL,
+    'is_currently_employed' NUMBER DEFAULT 1, /* in mySQL there is no bool values, so i menaged that it could be a number, but only 0 and 1 */
+    'salary' DECIMAL(7,2) DEFAULT NULL,
```

Comment:
salary and is_currently_employed are informations about contract and not employee

Reply:
But employee has a salary, salary could be null, if there was an employee who could not work any more